packages/forma-36-react-components/src/components/Checkbox/Checkbox.tsx

```diff
@@ -1,7 +1,8 @@
 import React, { Component } from 'react';
 import ControlledInput, { ControlledInputPropTypes } from '../ControlledInput';
 
-export type CheckboxProps = ControlledInputPropTypes & typeof defaultProps;
+export type CheckboxProps = ControlledInputPropTypes &
+  Partial<typeof defaultProps>;
```

Comment:
This still has the issue of the restrictive types from `defaultProps` overriding the declared types. Since the props types are the same as in the base component so you can do

```ts
// eslint-disable-next-line @typescript-eslint/no-empty-interface
export interface CheckboxProps extends ControlledInputPropTypes {};

const defaultProps: Partial<ControlledInputPropTypes>
```

The same for `RadioButton`, `SkeletonBodyText`, and `SkeletonDisplayText`.

Reply:
Oh - seems like I missed some components...